package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-cycle-middleware",
-  "version": "0.0.5",
+  "version": "0.1.0",
```

Comment:
@lmatteis I mean, versions are usually bumped in a separate commit so that it's easier to read the git log and see which version appeared where. Also it's usually project maintainers who cut the releases. Juuust my 2c here :)